packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ColorUtils.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+import androidx.annotation.VisibleForTesting
+
+private const val HEX_COLOR_INCLUDING_ALPHA_LENGTH: Int = 8
+
+internal fun resolveBackgroundColorAsHex(backgroundColor: Int): String {
+    val colorHexString = Integer.toHexString(backgroundColor)
+
+    // reorder the hex string from argb to rgba
+    return "#${reorderARGBtoRGBA(colorHexString)}"
+}
+
+@VisibleForTesting
+internal fun reorderARGBtoRGBA(hexString: String): String {
```

Comment:
maybe just `argbToRgba`? or `convertArgbToRgba`. I guess these names are more common, at least `convert` is the lingvo used in `android.graphics.Color`.